src/components/RelatedPhotoGallery/RelatedPhotoGallery.js

```diff
@@ -1,18 +1,29 @@
-import React, { PropTypes } from 'react';
+import React, { Component, PropTypes as T } from 'react';
+import { connect } from 'react-redux';
+import { getRelatedPhotos } from '../../actions/index';
 
-import Images from '../Images/Images';
-
-export default function RelatedPhotoGallery({
-  photos,
-}) {
-  // get('/photos/search', { geo: '43.6532,-79.3832,1km' }).then(res => console.log(res));
-  return (
+class RelatedPhotoGallery extends Component {
+  render() {
+    return (
     <div>
```

Comment:
At present, it isn't necessary because it's only the h3 tag but eventually there will be more lines in there so that's why I have the div set up.